test/e2e/runner/start.go

```diff
@@ -25,6 +25,9 @@ func Start(testnet *e2e.Testnet) error {
 		}
 		return false
 	})
+
+	initialHieght := testnet.InitialHeight
```

Comment:
Typo:
```suggestion
	initialHeight := testnet.InitialHeight
```

But also, can this please be moved down closer to its first use, e.g., just before Line 69?